rfcs/text/0000-project-completion-stage.md

```diff
@@ -0,0 +1,74 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# Project completion stage RFC
+
+## Summary
+
+This RFC proposes a process of releasing a contribution, or project, only after it has been completed by the DCP in order to maximize the value to the data consumer through reducing user confusion and minimizing errors in the final product.
+
+## Author(s)
+
+ [Jason Hilton](mailto:jahilton@stanford.edu)
+
+## Shepherd
+***Leave this blank.** This role is assigned by DCP PM to guide the **Author(s)** through the RFC process.*
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## Definitions
+
+A contribution, or project, is considered to be **version-complete** when the contributor confirms (by way of the wrangler) that they have provided all of the metadata and raw data that they plan to submit at the current time, the wrangler submits those to ingest, and the DCP has delivered every possible product, given current DCP capabilities.
+
+## Motivation
+
+**Display only completed data:**
+Any project that contains raw data with a corresponding analysis pipeline should be considered in progress until all downstream analysis is version-complete, validated, and available. Currently, a contributed project is made accessible to the public as soon as raw data & metadata are submitted. The trickling in of data can lead to data consumer confusion as to if/when more data will be added to the project. The proposed solution allows the DCP to complete the version of the project with DCP outputs before the project becomes publicly accessible.
+
+**More thorough data validation:**
+The downstream services of the DCP have the potential to uncover metadata and data errors. Viewing the version-complete project as the user will view it can also identify errors. A solution to this is a setting on projects allowing signed-in DCP members to access the data in order to perform a final ‘sanity check’.
+
+**Support data contributors (possibly 2nd phase):**
+Allowing data contributors to view the final product in their collaboration with the DCP before it is released adds yet another validation step (after all, who knows their data better?) and is a valuable step in good faith data sharing. The inclusion of a data lifecycle tracker on the project page keeps the contributor up-to-date on their submission and ensures a transparent process.
+
```

Comment:
While I like the idea of giving a contributor a chance to review we must also be careful we don't set an expectation that a contributor can block the release of data if they disagree with the results of our pipeline or just want to delay other people getting hold of the data. Such a release will need to have a timebox associated with it and passive acceptance being assumed at the end of that timebox

Reply:
Agreed, I have clarified that the contributor review is to be confined to the same week timebox as the internal DCP review. b175eb3

Reply:
Resolving this thread, please renew comments if I can address things further in the RFC.